enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/catchup/storecopy/LocalDatabase.java

```diff
@@ -84,6 +87,7 @@ public synchronized void start() throws Throwable
         log.info( "Starting with storeId: " + storeId );
 
         dataSourceManager.start();
+        watcherService.start();
```

Comment:
so far DS manager only responsible for neo data source and this new service is not really related to that. Like database health, for example, it's more service on the side and do not have anything to do to neo data source itself, so i would say that so far it's more related to local db abstraction.
(But maybe in future if worth to introduce service that will wrap all the db related services together to restart them in cases like this)